core/src/test/scala/org/apache/spark/util/collection/unsafe/sort/RadixSortSuite.scala

```diff
@@ -40,23 +41,37 @@ class RadixSortSuite extends SparkFunSuite with Logging {
   case class RadixSortType(
     name: String,
     referenceComparator: PrefixComparator,
-    startByteIdx: Int, endByteIdx: Int, descending: Boolean, signed: Boolean)
+    startByteIdx: Int, endByteIdx: Int, descending: Boolean, signed: Boolean, nullFirst: Boolean)
```

Comment:
For test cases in SortSuite, I am thinking of having another PR supporting null ordering for DataFrame/Dataset APIs and adding related test cases in SortSuite.